src/components/TopBar/TopBarMenu.vue

```diff
@@ -468,9 +467,16 @@ export default {
 			// If the current conversation is a break-out room and the user is not a moderator,
 			// also send request for assistance to the moderators.
 			if (this.userIsInBreakoutRoomAndInCall && !this.canModerate) {
-				this.$store.dispatch('requestAssistanceAction', {
-					token: this.token,
-				})
+				const hasRaisedHands = Object.keys(this.$store.getters.participantRaisedHandList).length !== 0
+				if (hasRaisedHands) {
+					return // Assistance is already requested by someone in the room
+				}
```

Comment:
When raising hand, it falls to that condition because `hasRaisedHands` is updated with the 'self' raised hand.

Reply:
Right :see_no_evil: Fixed:
```js
const hasRaisedHands = Object.keys(this.$store.getters.participantRaisedHandList)
	.filter(sessionId => sessionId !== this.$store.getters.getSessionId())
	.length !== 0
```